java/feathub-connectors/flink-connectors/flink-connector-prometheus/src/main/java/com/alibaba/feathub/flink/connectors/prometheus/PrometheusSinkFunction.java

```diff
@@ -193,6 +193,9 @@ public void updateMetric(RowData row, int index) throws IOException {
                 break;
             case MAP:
                 final MapData mapData = row.getMap(index);
+                if (mapData == null) {
```

Comment:
When would this happen? I am wondering if we should throw exception in this case.